packages/amazonq/.changes/next-release/Feature-2ca899db-fff6-469c-8294-f93fc965cfa7.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Enable SageMaker SSO user access token & Profile ARN to be used for accessing AmazonQ & CodeWhisperer features"
```

"CodeWhisperer" is no more
```suggestion
	"description": "Auth: Enable SSO access for SageMaker users."
```